runtime/services/catalog/migrator/metricsviews/metrics_views.go

```diff
@@ -74,12 +74,24 @@ func (m *metricsViewMigrator) Validate(ctx context.Context, olap drivers.OLAPSto
 
 	var validationErrors []*runtimev1.ReconcileError
 
+	dimensionNames := make(map[string]bool)
 	for i, dimension := range mv.Dimensions {
-		if _, ok := fieldsMap[strings.ToLower(dimension.Name)]; !ok {
+		if _, ok := dimensionNames[dimension.Name]; ok {
```

Comment:
Should we make this check case insensitive?

Reply:
Same applies to measures as well